### DISCUSSION TITLE
ip address 127.0.0.1 not working

### DISCUSSION THREAD

**Discussion by chesterflaps** [fuatakgun/eufy_security#767] [Q&A]
Hi all.

I am very new to HA in general so please be gentle.

I have followed the steps to get eufy security working but when I add the integration and the ip address of 127.0.0.1 is already inputted, it tells me it is incorrect. I have tried the ip address of my eufy homebase but that does not work either.

Does anyone have any ideas please?

Thank you.

**Comment by chesterflaps** [comment]
Ignore me, I got it working. I needed to add another eufy security add on.

**Reply by mohammad-shadmehr** [reply]
How did you resolve this? Where can I get the IP and port details for Eufy Security Web Socket?

**Reply by mohammad-shadmehr** [reply]
![image](https://user-images.githubusercontent.com/8295373/231738373-764e6d38-ed3a-4203-882d-9bb9fe6d99b2.png)

**Comment by KillaDreams** [comment]
please search issues before making posts, there have been a lot of duplicate issues/discussions the last few weeks

**Comment by KillaDreams** [comment]
Follow the instructions... Addon, HACS Repo, Integration
